src/lib/eth.js

```diff
@@ -279,5 +276,5 @@ try {
     .catch(e => logger.warn(`failed to connect${e}`));
 } catch (e) {
   logger.error('isConnected failed');
-  logger.warn('Ethereum node must be running for this module to work')
+  logger.warn('Ethereum node must be running for this module to work');
```

Comment:
may be nicer to output a hint that includes `npm run start:testrpc` or similar and also the value of the env var that specifies what eth rpc url it is using.